client/webserver/site/src/js/markets.ts

```diff
@@ -1148,6 +1150,17 @@ export default class MarketsPage extends BasePage {
     this.loadUserOrders()
   }
 
+  /*
+    setOrderFormTickers sets the base and quote tickers on the order form.
+  */
+  setOrderFormTickers (baseSym: string, quoteSym: string) {
+    const page = this.page
+    Doc.setText(page.priceBox, '[data-base-ticker]', baseSym)
+    Doc.setText(page.qtyBox, '[data-base-ticker]', baseSym)
+    Doc.setText(page.priceBox, '[data-quote-ticker]', quoteSym)
+    Doc.setText(page.mktBuyBox, '[data-quote-ticker]', quoteSym)
+  }
```

Comment:
We have `app().updateMarketElements`. Just run that on the `this.main` in `setMarket`.

Reply:
Oh okay.